### PR TITLE
Javascript methods

### PR DESCRIPTION
**Pull request by joshuapreciado**
Not finished

### CODE REVIEW

**Review by joshuapreciado — COMMENTED** [review]
JAVASCRIPT-METHODS

File: javascript-methods/main.js [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+console.log('lastBook:', lastBook);
+
+var firstBook = library.shift();
+console.log('firstBook:', firstBook);
+
+library.push(js);
+
+library.unshift(css);
+
+library.splice(2);
```
Comment (BrettAlbright) [inline javascript-methods/main.js]: There should be 2 arguments here, the first indicates where to start removing elements, and the second one is how many to remove. You want to make sure the output includes the CSS and JS books.

**Review by Zhiwutian — CHANGES_REQUESTED** [review]
small update needed!

**Review by Zhiwutian — APPROVED** [review]
👍🏼

**Pull request merged**